### ISSUE TITLE
Support new method visibility syntax

### ISSUE THREAD

**Issue by ayqazi** [zenspider/enhanced-ruby-mode#67]
Hi,

Ruby 2.1 now allows you to do this:

```
private def foo
  puts "bar"
end
```

enh-ruby-mode indents this as:

```
private def foo
          puts "bar"
        end
```

I intend to adopt this immediately, can enh-ruby-mode be changed to indent like the former example?

Thanks

**Comment by stardiviner** [comment]
-1

**Comment by sockmonk** [comment]
I don't like hunting above a method in the file for `private` either, so I've always used `private :foo` or `private_class_method :foo`. Maybe that would work as a compromise? (without having to change enh-ruby)

**Comment by zenspider** [comment]
What's the status of this?

**Comment by ayqazi** [comment]
I'm afraid I don't use Ruby that heavily any more, and am no longer coding in Rails, so I'm afraid I won't be able to invest time doing this.

**Comment by zenspider** [comment]
fair enough. I'll try to get a failing test written at some point but I'll leave this open if someone wants to take a whack at it. It isn't a priority for me, personally.

**Comment by zenspider** [comment]
Nobody is touching this so I don't think it is a real priority. Closing to freshen up this project.

**Comment by knu** [comment]
I've just come up with a PR: #110.

**Comment by stardiviner** [comment]
Seems this feature request always keep a low requirement.

**Comment by knu** [comment]
Actually, the original ruby-mode indents annotated def's _that_ way, so I guess those who prefer the style simply stick with ruby-mode.

**Comment by zenspider** [comment]
This should be fixed by @knu's #110

**Comment by stardiviner** [comment]
Thanks @knu @zenspider
